src/main/cc/maze/cell/cell.c

```diff
@@ -0,0 +1,23 @@
+#include "cell.h"
+
+Cell Cell::Cell(int x_loc, int y_loc) {
+	x = x_loc;
+	y = y_loc;
+	visited = false; 
+}
+
+int Cell::GetLocationX(void) {
```

Comment:
I also think it might be better to call these GetXLocation and GetYLocation as to me that is more readable as that is how you would use it in a sentence, like I want to get the x location not I want to get the location x.